app/models/task_repository.rb

```diff
@@ -2,21 +2,33 @@ module TaskRepository
   class << self
 
     def find_by_id(task_id)
-      task_with_taggings.find_by(id: task_id)
+      Task.includes(:taggings, { assignments: :user })
+        .find_by(id: task_id)
     end
 
-    def all_by_author_id(author_id)
-      task_with_taggings.where(author_id: author_id).order(:id)
+    def all_uncompleted_by_author_id(author_id)
+      Task.includes(:completed_task, :taggings, { assignments: :user })
+        .where(author_id: author_id)
+        .where(completed_tasks: { id: nil })
+        .order('tasks.id, taggings.id')
     end
 
-    def all_by_tag(tag)
-      Task.joins(:taggings).where(taggings: { tag: tag }).preload(:taggings)
+    def all_completed_by_author_id(author_id, from_date: nil)
+      base = Task.includes(:completed_task, :taggings)
+               .where(author_id: author_id)
```

Comment:
Style/MultilineOperationIndentation: Align the operands of an expression in an assignment spanning multiple lines.